### DISCUSSION TITLE
Added: C# design notes for Feb 25th, Feb. 27th, and March 4th, 2019

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2326] [General]
Feb 25th: https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-02-25.md

Feb 27th: https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-02-27.md

March 4th: https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-03-04.md

**Comment by dsaf** [comment]
> People who used the feature for overall code hygiene were more excited about using the feature

Loved that one :). In my experience, people who see compiler errors/warnings as a nuisance are not interested in new versions of C# at all.

**Comment by yaakov-h** [comment]
all your `base()` are belong to us
